resources/warnings.json

```
@@ -14938,7 +14938,7 @@
     "versions": [
       {
         "lastVersion": "2.8.1",
-        "pattern": ".*"
+        "pattern": "([1].*)|([2].[0-8].*)"
```

This will match all versions starting with `1`, e.g. a potential future `10`.

I recommend instead:
```suggestion
        "pattern": "(1|2[.][0-8])(|[.-].+)"
```